packages/rn-tester/js/examples/Layout/LayoutAnimationExample.js

```diff
@@ -48,19 +51,65 @@ function shuffleArray(array: Array<ExampleViewSpec>) {
   return array;
 }
 
+type OptionBarPropType = {|
+  options: Array<string | number>,
+  callback: Function,
+  selectedOption: string,
+|};
+
+class OptionBar extends React.Component<OptionBarPropType> {
+  render() {
+    return (
+      <ScrollView
+        style={styles.choiceContainer}
+        horizontal
+        showsHorizontalScrollIndicator={false}>
+        {this.props.options.map(option => (
+          <TouchableOpacity
+            onPress={() => this.props.callback(option)}
+            key={option}
+            style={[
+              styles.optionStyle,
+              {
+                backgroundColor:
+                  this.props.selectedOption === option ? '#ccc' : 'white',
```

Comment:
@AnshG714 , try using hex code instead of white for consistency.

Reply:
Done!